images/openshift-enterprise-ansible-operator.yml

```diff
@@ -27,3 +27,5 @@ owners:
 - jesusr@redhat.com
 - jlanford@redhat.com
 - fabian@redhat.com
+non_shipping_repos:
+- rhel-8-server-ose-rpms
```

Comment:
This doesn't seem right.  ansible-operator requires ansible-runner and a bunch of python3-* packages that are (in the context of openshift) only available from this repo.

Reply:
`rhel-8-server-ose-rpms` is added to `non_shipping_repos` by the script because CVP claims it is redundant. The script just reads whatever the latest CVP test results but I am not sure how CVP works though... Maybe we can defer this work until 4.6 GA?

CC @sosiouxme

Reply:
see for example http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-product-test/openshift-enterprise-ansible-operator-container-v4.6.0-202010011936.p0/9bf0f895-3eba-40ad-ad89-c78a150bbb2c/sanity-tests-optional-results.json where the non-x86_64 content sets are declared redundant.

If I look at the [ppc64le logs for that build](http://download.eng.bos.redhat.com/brewroot/packages/openshift-enterprise-ansible-operator-container/v4.6.0/202010011936.p0/data/logs/ppc64le-build.log) there certainly seems to be a great deal installed from this (arch-specific) repo.

It seems likely to be a CVP bug.

Reply:
I filed https://projects.engineering.redhat.com/browse/CLOUDBLD-2865

We can defer fixes until the CVP results seem correct.